dirty_cat/target_encoder.py

```diff
@@ -71,13 +102,22 @@ class TargetEncoder(BaseEstimator, TransformerMixin):
     def __init__(self,
                  categories='auto',
                  clf_type='binary-clf',
-                 dtype=np.float64, handle_unknown='error',
-                 handle_missing=''):
+                 dtype=np.float64,
+                 handle_unknown='error',
+                 handle_missing='',
+                 cross_val=False,
+                 n_folds=5,
+                 n_inner_folds=3,
+                 random_state=1):
```

Comment:
Scikitlearn seems to usually default to `random_state=None` (different values when rerun)

Reply:
Yes, you are right, that is better. Thanks!